sea-orm-macros/Cargo.toml

```diff
@@ -29,4 +29,7 @@ sea-orm = { path = "../", features = ["macros"] }
 serde = { version = "1.0", features = ["derive"] }
 
 [features]
+default = ["derive"]
 postgres-array = []
+derive = ["bae"]
+strum = []
```

Comment:
Same, I think we just ship it by default

Reply:
For this, dependency are conditionally enabled when a feature is being enabled.

This should be fine. If we decided to keep `derive` and `strum` as it's now.